core/app/services/uffizzi_core/deployment_service.rb

```diff
@@ -29,6 +29,25 @@ def create_from_compose(compose_file, project, user)
       deployment_form
     end
 
+    def update_from_compose(compose_file, project, user, deployment_id)
+      deployment_attributes = ActionController::Parameters.new(compose_file.template.payload)
+
+      deployment_form = UffizziCore::Api::Cli::V1::Deployment::UpdateForm.new(deployment_attributes)
+      deployment_form.assign_dependences!(project, user)
+      deployment_form.compose_file = compose_file
+
+      if deployment_form.valid?
+        deployment = UffizziCore::Deployment.find(deployment_id)
```

Comment:
It should be in the controller

Reply:
resolved in this MR https://github.com/UffizziCloud/uffizzi_app/pull/124